gempython/utils/db/settings.py

```diff
@@ -0,0 +1,17 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "gempython.utils.db.settings.local")
+SECRET_KEY = 'ef7$_)r0$#(x)ovd^m_6_bevsonkaxalvrhds9@ilb_p0o&-xn'
+
+INSTALLED_APPS = [
+    'gempython.utils.ldqm_db',
+    ]
+DATABASES = {
+    'default': {
+        'ENGINE':  'django.db.backends.mysql',
+        'NAME':     'ldqm_tif_qc8_db',
+        'PORT':     3306,
+        'HOST':     'cosmicstandtif',
+        'USER':     'gemdaq',
+        'PASSWORD': 'gemdaq',
+    }
+}
```

Comment:
hmmm... The raw content shows no duplication
https://raw.githubusercontent.com/mexanick/cmsgemos/7293d28e2a30c16b6ee3a39846d160325438f832/gempython/utils/db/settings.py